spock-core/src/main/java/org/spockframework/runtime/extension/builtin/PendingFeatureBaseInterceptor.java

```diff
@@ -25,6 +27,7 @@ protected AssertionError featurePassedUnexpectedly() {
   }
 
   protected AssumptionViolatedException assumptionViolation() {
-    return new AssumptionViolatedException("Feature not yet implemented correctly.");
+    return new AssumptionViolatedException("Feature not yet implemented correctly."
+      + ("".equals(reason) ? "" : "Reason: "+reason));
```

Comment:
If I'm not mistaken there's no space after the period now. It should be `" Reason: "`. Moreover spaces around the `+` would be more readable IMHO.